monkey/infection_monkey/network/network_scanner.py

```diff
@@ -79,7 +78,10 @@ def get_victim_machines(self, scan_type, max_find=5, stop_callback=None):
         for net_range in self._ranges:
             LOG.debug("Scanning for potential victims in the network %r", net_range)
             for ip_addr in net_range:
-                victim = VictimHost(ip_addr)
+                if hasattr(net_range, 'domain_name'):
```

Comment:
So every machine from `SingleIpRange` will have a domain_name? because `string_to_host` will always return a `domain_name` trait

Reply:
Yes, you can see it at SingleIpRange __init__() (network_range.py : 115).
This check is for those _ranges that do not have this attribute (objects of other classes).

Reply:
Yeah, what I thought happens here. If we give a range object of '192.168.1.1' then it will have a domain name of ''. This means that the VictimHost will have a domain name and in the display, you'll show it as ''.